src/expressions/splitExpression.ts

```diff
@@ -200,7 +201,14 @@ export class SplitExpression extends ChainableExpression implements Aggregate {
 
   public getSelectSQL(dialect: SQLDialect): string[] {
     return this.mapSplits((name, expression) => {
-      if (['IP', 'SET/IP'].includes(expression.type)) {
+      if (
+        expression instanceof SqlRefExpression &&
+        ['IP', 'SET/IP'].includes(expression.type) &&
+        !(
+          expression.sql.toLowerCase().includes('ip_search') ||
+          expression.sql.toLowerCase().includes('ip_match')
```

Comment:
What if the column name is `ip_search` or `ip_match` but it doesn't actually use those functions?